ui/app/components/page/userpass-reset-password.hbs

```diff
@@ -0,0 +1,54 @@
+<PageHeader as |p|>
+  <p.levelLeft>
+    <h1 data-test-title class="title is-3">
+      Reset password
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+{{#if this.successful}}
+  <div class="box is-sideless is-fullwidth is-marginless">
+    <Hds::Alert @type="inline" @color="success" data-test-reset-password-success as |A|>
+      <A.Title>Password set successfully</A.Title>
+      <A.Button @text="Reset form" @color="secondary" {{on "click" this.reset}} data-test-reset-reset-password />
+    </Hds::Alert>
+  </div>
+{{else}}
+  <form {{on "submit" (perform this.updatePassword) on="submit"}}>
+    <div class="box is-sideless is-fullwidth is-marginless">
+      <Hds::Alert @type="inline" class="has-bottom-margin-m" as |A|>
+        <A.Title>Current user</A.Title>
+        <A.Description data-test-current-user-banner>You are updating the password for
+          <strong>{{@username}}</strong>
+          on the
+          <strong>{{@backend}}</strong>
+          auth mount.</A.Description>
+      </Hds::Alert>
+      <FormFieldLabel for="reset-password" @label="New password" />
+      <MaskedInput
+        id="reset-password"
+        @name="reset-password"
+        @value={{this.newPassword}}
+        @onChange={{fn (mut this.newPassword)}}
+      />
+    </div>
+    <div class="field is-grouped-split box is-fullwidth is-bottomless">
+      <div class="field is-grouped">
```

Comment:
To minimize `<div>` usage I've been using `<hr>` elements to separate form buttons. Just offering as an alternative
```js
<hr class="has-background-gray-200 has-top-margin-l" />
<Hds::ButtonSet>
....
```